[minor] Custom command ##Flags annotation panic

**Describe the bug**

In launch command, use 
```
## Flags: [{"Name":"phpmyadmin","Shorthand":"p","Usage":"Launch phpMyAdmin", "Type":"string"},{"Name":"mailhog","Shorthand":"m","Usage":"Launch mailhog","type":"string"}]
```

```
$ ddev
panic: Error implementation of DefValue validation missing for type 'string'

goroutine 1 [running]:
github.com/drud/ddev/cmd/ddev/cmd.(*defValueValue).validate(0xc0001ec1e0, 0xc0000aa7ba, 0x6, 0x0, 0x0)
	/workdir/cmd/ddev/cmd/flags.go:231 +0x3bf
github.com/drud/ddev/cmd/ddev/cmd.(*Flag).validateFlag(0xc0001ec1a0, 0xc00016d710, 0xc00016d708, 0x1, 0xc00016d900)
	/workdir/cmd/ddev/cmd/flags.go:267 +0x245
github.com/drud/ddev/cmd/ddev/cmd.(*Flags).validateFlags(0xc00016dd18, 0xc0000b5d60, 0xa0, 0x1742600)
	/workdir/cmd/ddev/cmd/flags.go:298 +0x237
github.com/drud/ddev/cmd/ddev/cmd.(*Flags).LoadFromJSON(0xc00016dd18, 0xc000160847, 0xa0, 0x5, 0xc0001e02e8)
	/workdir/cmd/ddev/cmd/flags.go:329 +0xd7
github.com/drud/ddev/cmd/ddev/cmd.addCustomCommands(0x1d9ef20, 0x0, 0x1889565)
	/workdir/cmd/ddev/cmd/commands.go:122 +0x13a5
github.com/drud/ddev/cmd/ddev/cmd.init.31()
	/workdir/cmd/ddev/cmd/root.go:173 +0x175

I was successful with `## Flags: [{"Name":"phpmyadmin","Shorthand":"p","Usage":"Launch phpMyAdmin"},{"Name":"mailhog","Shorthand":"m","Usage":"Launch mailhog"}]`

But I did struggle. Between json (which is always terrible) and difficult error messages...  And I didn't remember that the whole purpose of this was to just present help.

--- a/cmd/ddev/cmd/flags.go
+++ b/cmd/ddev/cmd/flags.go
@@ -223,6 +223,8 @@ func (v *defValueValue) validate(aType typeValue) error {
 		*v = defValueValue(strconv.FormatBool(false))
 	case FtCount, FtDuration, FtFloat32, FtFloat64, FtInt, FtInt8, FtInt16, FtInt32, FtUint, FtUint8, FtUint16, FtUint32:
 		*v = defValueValue(strconv.FormatInt(0, 10))
+	case FtString:
+		*v = ""
 	case ftTest3: // used for testing only
 		*v = ""
 	default:

